src/org/labkey/test/tests/FileBasedPipelineTest.java

```diff
@@ -72,6 +73,28 @@ public void startTest()
         goToProjectHome();
     }
 
+    @Test
+    public void testScriptBasedFilewatchers()
```

Comment:
File watchers require the premium module. This test case need to be in the `premium` module or it will cause community test runs to fail.

Reply:
Ah ok! I'll move it over.